README.md

```diff
@@ -41,6 +43,21 @@ flowchart TB
 
 ```
 
+## Getting started
+
+Clone the repository with its submodules: 
+
+````
+git clone --recurse-submodules https://github.com/prebid/paf-mvp-implementation.git
+````
+
+Memo: if you forget the submodules option:
+
+```
+git submodule update --init --recursive
```

Comment:
[FAR] add some instructions for when we want to simply **_update_** the submodule.

Reply:
Done.
Note that the `update` can be misunderstood when speaking of submodules. I guess by updating, you mean changing the HEAD of the submodule. I _updated_ (:D) the CONTRIBUTING.md